src/calculateRentalCost.js

```diff
@@ -5,6 +5,28 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const daylyRent = 40;
+  let amount = days * daylyRent;
+
+  if (days <= 3) {
+    amount = days * daylyRent;
+
+    // return amount;
+  }
```

Comment:
The condition 'days <= 3' is not necessary since the amount is already calculated before the if statements. You can remove this block.